CHANGELOG.md

```diff
@@ -9,12 +9,18 @@ Changes since last non-beta release.
 
 _Please add entries here for your pull requests that are not yet released._
 
-### Breaking Changes
-- Remove support & testing for Webpacker 3/4.
-- Add support & testing for Shakapacker v6
+## Breaking Changes
+### Removed
+- Support & testing for Webpacker 3/4.
+- WebpackManifestContainer, which searched for assets in the webpack manifest
+### Added
+- Support & testing for Shakapacker v6
+- SeparateServerBundleContainer, which expects a single server bundle file & does not use the webpack manifest at all.
+### Usage
 - Requires separate compilations for server & client bundles (see [Webpack config](https://github.com/reactjs/react-rails/tree/master/test/dummy/config/webpack))
-- Replaces WebpackManifestContainer, which searched for assets in the webpack manifest, with SeparateServerBundleContainer, which expects a single server bundle file & does not use the webpack manifest at all.
 - All in one PR! #1274
+### Dependencies
```

Comment:
If we want to follow our convention, we need to add each change in the appropriate category and then add the PR number. It is fine to repeat the PR number in multiple entries.

This way, we won't have any line saying "All in one PR..."

Also since this is a changelog for the release, we shouldn't both say add support for Shakapacker 6 and also add support for Shakapacker 7. We basically drop support for Shakapacker 6. We jump to Shakapacker 7.